source/MRViewer/MRUIStyle.ipp

```diff
@@ -257,6 +257,10 @@ bool slider( const char* label, T& v, const U& vMin, const U& vMax, UnitToString
                 ret = true;
                 detail::markItemEdited( ImGui::GetItemID() );
             }
+
+            if ( *elemMin < *elemMax ) // sometimes ImGui does not clamp it, so make sure that value is clamped
```

Comment:
1. Please move this up to line 239 above, to prevent it from flashing incorrect value for one frame.
2. I think we should also add `flags & ...AlwaysClamp` to the condition.

Reply:
1. I think its better to clamp just before returning
2. Done